src/cmd/internal.go

```diff
@@ -196,6 +196,27 @@ var createPackageRegistryToken = &cobra.Command{
 	},
 }
 
+var updateGiteaPVC = &cobra.Command{
+	Use:   "update-gitea-pvc",
+	Short: lang.CmdInternalUpdateGiteaPVCShort,
+	Long:  lang.CmdInternalUpdateGiteaPVCLong,
+	Run: func(cmd *cobra.Command, args []string) {
+		c := cluster.NewClusterOrDie()
```

Comment:
should this cluster connection be passed down to the UpdateGiteaPVC() method instead of creating another connection inside that method?

Reply:
The function doesn't need a git config so it doesn't need to be a receiver and you don't need to load state here - I would drop this cluster connection and just make it a function in the `git` package instead of a receiver.